app/views/local_authorities/index.html.erb

```diff
@@ -13,7 +13,13 @@
       locals: {placeholder: "Filter list of local authorities"} %>
     </thead>
     <tbody>
-      <%= render @authorities %>
+      <% @authorities.each do |authority| %>
+        <tr>
+          <td class="name">
+            <%= link_to authority.name, local_authority_path(authority.slug), class: 'js-open-on-submit' %>
```

Comment:
Ah!  I did a search on github but could only find it in views; forgot to look in govuk admin template for the js implementation :(.  Thanks!